lib/mayaUsd/undo/UsdUndoStateDelegate.cpp

```diff
@@ -23,17 +23,23 @@
 PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace {
+
 void copySpecAtPath(const SdfAbstractData& src, SdfAbstractData* dst, const SdfPath& path)
 {
+    // create a new spec at a path with the given specType
     dst->CreateSpec(path, src.GetSpecType(path));
 
+    // get the list of tokens
     const std::vector<TfToken>& tokens = src.List(path);
 
+    // set the value of dst at the given path and a fieldName
     for (const auto& token : tokens) {
         dst->Set(path, token, src.Get(path, token));
     }
 }
 
+// This class is used to copy specs from source SdfAbstractData contrainer
```

Comment:
@ppt-adsk  I have `testungroup.py ` and `UsdUndoUngroupCmd[*.h][*.cpp] ` wrapped in `CMAKE_UFE_V3_FEATURES_AVAILABLE` since these are UFE3 features.  See 91e77bc1664925763db75b4dad165bded244c179